risc0/build/README.md

```diff
@@ -53,4 +53,8 @@ be named `methods::MULTIPLY_ELF`. These are included at the beginning
 of the host-side code:
 ```text
 use methods::{MULTIPLY_ELF, MULTIPLY_ID};
-```
\ No newline at end of file
+```
+
+# Crate Feature Flags
+
+There are currently no feature flags available for `build`.
```

Comment:
I'm not sure we even need this section for a crate with no feature flags (but probably it doesn't cause any harm, either).

Reply:
Ah, I was thinking it would disambiguate between "I forgot to add this" and "These are not present" but if it's odd enough to stick out I'll pull it